server/plugin_release.go

```diff
@@ -500,12 +502,12 @@ func downloadAsset(ctx context.Context, client *GithubClient, owner, repositoryN
 func getPluginAsset(ctx context.Context, githubClient *GithubClient, owner, repo, tag string) (*github.ReleaseAsset, error) {
 	LogInfo("Checking if the release asset is available")
 
-	ctx, cancel := context.WithTimeout(ctx, 20*time.Minute)
+	ctx, cancel := context.WithTimeout(ctx, pluginAssetTimeout)
 	defer cancel()
 
 	for {
 		// Using timer to avoid memory leaks
-		timer := time.NewTimer(30 * time.Second)
+		timer := time.NewTimer(pluginAssetTimeout)
```

Comment:
This is the wrong timeout to change, right? (Otherwise builds will take at least 50 minutes before we check.)

Reply:
whoops, you're right!  Aggressive refactoring.
Not sure if we need the `const` now, but kept it to make it easy to make changes in the future.